src/bin/stratisd.rs

```diff
@@ -75,15 +69,21 @@ fn main() {
         .get_matches();
 
     if matches.is_present("debug") {
-        unsafe { DEBUG = true }
-    };
+        let mut builder = LogBuilder::new();
```

Comment:
We should call one or the other of ```filter()```, ```parse()``` since they do the same thing, AFAIU. I do not like ```parse()``` because it is more heavyweight, as it has to parse a string by means of regular expressions. ```filter()``` is better, because more programmatic. I think ```parse()``` is more appropriate if someone had passed in those strings that is parses from the command line.